README.md

```diff
@@ -11,7 +11,7 @@ docker-compose up -d
 docker-compose run --rm --user "${UID}:${GID}" web bash -c 'cd /var/www && composer install'
 
 # Optional:  Load data
-docker-compose exec -T db bash -c 'mysql -u "$MARIADB_USER" -p"$MARIADB_PASSWORD" "$MARIADB_DATABASE"' < data.sql
+docker-compose exec -T db bash -c 'mariadb -u"$MARIADB_USER" -p"$MARIADB_PASSWORD" "$MARIADB_DATABASE"' < data.sql
```

Comment:
needed to change this to mariadb from mysql. mysql did not work for me. It may be due to [this update from MariaDB v. 11](https://mariadb.com/kb/en/mariadb-11-0-1-release-notes/#docker-official-images):
"11.0, unlike previous version, no longer includes mysql named compatible executable symlinks inside the container. "